core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -88,89 +85,125 @@ public ClientLogger(String className) {
     }
 
     /**
-     * Sets the logger to the verbose logging level.
+     * Logs a format-able message that uses {@code {}} as the placeholder at {@code verbose} log level
      *
-     * @return Updated ClientLogger if verbose is enabled.
-     */
-    public ClientLogger asVerbose() {
-        return asLevel(VERBOSE_LEVEL);
-    }
-
-    /**
-     * Sets the logger to the info logging level.
+     * <p><strong>Code Samples</strong></p>
+     * <p>
+     * Logging a message at verbose log level
+     * <pre>
+     * ClientLogger logger = new ClientLogger(Example.class);
+     * logger.logAsVerbose("A format-able message. Hello, {}", name);
```

Comment:
Now would be a good time to extract these code samples out into Java files.

Reply:
Will use code snippets instead.